rust/data/presets.json

```diff
@@ -229,29 +225,31 @@
       "canTrickyDashJump",
       "canCWJ",
       "canHBJ",
-      "canUnmorphBombBoost",
+      "canDoubleBombJump",
```

Comment:
I'm surprised about this one. Using it efficiently or for a large climb could be a bit tricky. One place that this is used is for getting the Blue Brinstar ETank with a jump into IBJ, which I'm not thinking should be Expert.

Reply:
Very few people are comfortable with double bomb jumps.

Ceiling Etank is closer to a hard single IBJ techwise imo.  But there is another way we could write it that works.  3 single bombs following a jump and an unmorph.  And that isn't a unmorphBombBoost but instead a hitboxExtend.